softlayer/vm/softlayer_vm.go

```diff
@@ -32,6 +34,8 @@ type SoftLayerVM struct {
 	sshClient util.SshClient
 
 	logger boshlog.Logger
+
+	timeOutTestTag bool
```

Comment:
No. So This is not the way to do this. Instead, inject the actual timeout as a parameter when creating the VM. That way you can provide the right value when creating the VM in test with the fake client.

You can still keep the default timeout as public constant in the SoftLayerVM and use that as needed when creating a new VM in the create VM implementation.